examples/with-emailpassword-vercel/src/App.js

```diff
@@ -10,13 +10,13 @@ import SessionExpiredPopup from "./SessionExpiredPopup";
 
 export function getApiDomain() {
     const apiPort = process.env.REACT_APP_API_PORT || 3001;
-    const apiUrl = window.location.hostname || `http://localhost:${apiPort}`;
+    const apiUrl = process.env.REACT_APP_VERCEL_URL !== undefined ? window.location.hostname  : `http://localhost:${apiPort}`;
```

Comment:
In the frontend `process.env.REACT_APP_VERCEL_URL` will always be `undefined`... Just make it `const apiUrl = window.location.hostname || `http://localhost:${apiPort}`